opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -73,6 +96,28 @@ namespace Opm {
         return fun::map( mknode, wnames );
     }
 
+    static inline std::vector< ERT::smspec_node > keywordW(
```

Comment:
I've taken note and put them in my TODO when I touch this file sometime soon. I don't think it should block this PR though.